aspnetcore/host-and-deploy/scaling-aspnet-apps/scaling-aspnet-apps.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Scaling ASP.NET Core Apps on Azure
+author: alexwolfmsft
+description: Learn how to horizontally scale ASP.NET Core apps on Azure and address common architectural challenges
+ms.author: alexwolf
+ms.custom: mvc
+ms.date: 08/25/2022
+---
+# Deploying and scaling an ASP.NET Core app on Azure Container Apps
+
+Web applications deployed to Azure should be dynamically scalable in order to leverage various benefits of the cloud, such as elasticity. ASP.NET Core apps meet these requirements, but developers must consider certain architectural patterns and configurations to ensure success. This tutorial demonstrates how to deploy a scalable Razor Pages app to Azure Container Apps by completing the following tasks:
+
+1. [Setup the sample project](#1-setup-the-sample-project)
+1. [Deploy the app to Azure Container Apps](#2-deploy-the-app-to-azure-container-apps)
+1. [Scale and troubleshoot the app](#3-scale-and-troubleshoot-the-app)
+1. [Create the Azure Services](#4-create-the-azure-services)
+1. [Connect the Azure services](#5-connect-the-azure-services)
+1. [Configure and redeploy the app](#6-configure-and-redeploy-the-app)
+
+In some cases, simple ASP.NET Core apps are able to scale without special considerations. However, some apps require additional configurations to utilize certain framework features or architectural patterns, including the following:
```

Comment:
simple is banned from this repo. Please look at the changes I made rather than tossing all my work.

Reply:
Thanks for the comments, this is my first time working in this repo. I moved the updated version of the article into this branch and am still updating it as a draft PR - I started integrating your recommendations and will go through the rest as well. I opened a draft PR to generate a preview as I work on this - is this the desired workflow for this repo?